pennylane/ops/op_math/__init__.py

```diff
@@ -54,7 +54,7 @@
 from .adjoint_class import Adjoint
 from .adjoint_constructor import adjoint
 from .controlled_class import Controlled, ControlledOp
-from .exp import exp, Exp
+from .exp import exp, Exp, Evolution
```

Comment:
Add Evolution to the list of symbolic classes above so that it shows up in documentation 👍🏼